src/app/text-templating/text-templating.ts

```diff
@@ -8,22 +8,22 @@ import { referenceDate, pastDate } from "./dateHandler";
 import { partial } from "./partial";
 import { switcher } from "./switcher";
 
-export const shadowTargetProperties : any[] = []
+export const shadowTargetProperties: any[] = []
```

Comment:
Is this resulting from the execution of the linter?

Reply:
It was probably just the "Format Document" option from VSCode